src/JupyterLibrary/clients/jupyterlab/Selectors.robot

```diff
@@ -31,6 +31,11 @@ ${JLAB XP NB FRAG}    ${JLAB XP MAIN AREA FRAG}\[contains(@class, 'jp-NotebookPa
 ${JLAB XP NB TOOLBAR FRAG}    [contains(@class, 'jp-NotebookPanel-toolbar')]
 ${JLAB XP NB TOOLBAR}    //div${JLAB XP NB TOOLBAR FRAG}
 ${JLAB XP BUSY KERNEL}    //*[local-name() = 'div' and conttains(@class, 'jp-FilledCircleIcon' or (local-name() = 'svg' and contains(@data-icon, 'ui-components:circle-filled')))]
+${CELL XP INPUT}  div[contains(@class, p-Cell-inputWrapper)]/div[contains(@class,"jp-Cell-inputArea")]
```

Comment:
these should likely conform to the other naming conventions. Also, `p-Cell-inputWrapper` probably needs quoting (looks like single quoting throughout, but that might just be the review window).

Reply:
solved